fees/swing/index.ts

```diff
@@ -0,0 +1,104 @@
+import { CHAIN } from "../../helpers/chains";
+import { httpGet } from "../../utils/fetchURL";
+import { FetchOptions, SimpleAdapter } from "../../adapters/types";
+import { getUniqStartOfTodayTimestamp } from "../../helpers/getUniSubgraphFees";
+
+const chains: Record<string, string> = {
+    [CHAIN.SOLANA]: 'solana',
+    [CHAIN.ETHEREUM]: 'ethereum',
+    [CHAIN.BSC]: 'bsc',
+    [CHAIN.AVAX]: 'avalanche',
+    [CHAIN.POLYGON]: 'polygon',
+    [CHAIN.ARBITRUM]: 'arbitrum',
+    [CHAIN.ARCHWAY]: 'archway-1',
+    [CHAIN.B2_NETWORK]: 'b2-network',
+    [CHAIN.BASE]: 'base',
+    [CHAIN.BITCOIN]: 'bitcoin',
+    [CHAIN.BITLAYER]: 'bitlayer',
+    [CHAIN.BLAST]: 'blast',
+    [CHAIN.BOB]: 'bob',
+    [CHAIN.CORE]: 'core-blockchain',
+    [CHAIN.COSMOS]: 'cosmoshub-4',
+    [CHAIN.FANTOM]: 'fantom',
+    [CHAIN.XDAI]: 'gnosis',
+    [CHAIN.GRAVITY]: 'gravity',
+    [CHAIN.INJECTIVE]: 'injective-1',
+    [CHAIN.LINEA]: 'linea',
+    [CHAIN.MANTA]: 'manta-pacific',
+    [CHAIN.MANTLE]: 'mantle',
+    [CHAIN.METIS]: 'metis',
+    [CHAIN.MODE]: 'mode',
+    [CHAIN.MOONBEAM]: 'moonbeam',
+    [CHAIN.MORPH]: 'morph',
+    [CHAIN.CELESTIA]: 'cataclysm-1',
+    [CHAIN.OPTIMISM]: 'optimism',
+    [CHAIN.OSMOSIS]: 'osmosis-1',
+    [CHAIN.SCROLL]: 'scroll',
+    [CHAIN.TAIKO]: 'taiko',
+    [CHAIN.WORLD_CHAIN]: 'world-chain',
+    [CHAIN.ZKSYNC]: 'zksync-era',
+};
+
+const fetchVolume = async (_t: any, _b: any, options: FetchOptions) => {
```

Comment:
version:2 adapters have only one param, options

We are trying to use v2 for all new adapters